docs/getting_started/hello_world_tgb.py

```diff
@@ -0,0 +1,58 @@
+import taipy as tp
+from taipy import Config, Core, Gui
+import taipy.gui.builder as tgb
+
+
+################################################################
+#            Configure application                             #
+################################################################
+def build_message(name):
+    return f"Hello {name}!"
+
+
+# A first data node configuration to model an input name.
+input_name_data_node_cfg = Config.configure_data_node(id="input_name")
+# A second data node configuration to model the message to display.
+message_data_node_cfg = Config.configure_data_node(id="message")
+# A task configuration to model the build_message function.
+build_msg_task_cfg = Config.configure_task("build_msg", build_message, input_name_data_node_cfg, message_data_node_cfg)
+# The scenario configuration represents the whole execution graph.
+scenario_cfg = Config.configure_scenario("scenario", task_configs=[build_msg_task_cfg])
+
+################################################################
+#            Design graphical interface                        #
+################################################################
+
+input_name = "Taipy"
+message = None
+
+
+def submit_scenario(state):
+    state.scenario.input_name.write(state.input_name)
+    state.scenario.submit()
+    state.message = scenario.message.read()
+
+with tgb.Page() as page:
+    tgb.text("Name:")
+    tgb.input("{input_name}")
```

Comment:
I suspect that, when https://github.com/Avaiga/taipy/pull/635 is merged, that can be written:
`tgb.input(input_name)`

If I were you, I would give this a shot.